archetypes/oci-mp/src/main/resources/spec/README.md

```diff
@@ -0,0 +1,5 @@
+# Helidon OCI Archetype OpenAPI Specification Document
```

Comment:
I would remove "Helidon OCI Archetype" and leave this more generic (to minimize cruft users have to remove/modify when they change).

Reply:
Will fix README in upcoming PR along with documentation.